src/templates/pages/home.html

```diff
@@ -1,44 +1,345 @@
 {% extends "base.html" %}
 
 {% load staticfiles %}
+{% load i18n %}
 
 {% block top_div_extra_class %}jumbotron{% endblock %}
 
 {% block top_div_extra_content %}
 
     <div class="ui text container">
-        {#            <h1 class="ui inverted header">#}
-        {#                Codalab Competitions#}
-        {#            </h1>#}
         <img id="jumbo_logo" src="{% static "img/codalab-logo-futuraLT-light2.png" %}">
         <h2>Accelerating reproducible computational research.</h2>
-        {#            <div class="ui huge primary button">Get Started <i class="right arrow icon"></i></div>#}
-
-        <a id="style_button" href="{% url "pages:search" %}" class="pages ui huge button"><i
-            class="left arrow icon"></i> Compete</a>
-        <a id="style_button" href="{% url "pages:organize" %}" class="pages ui huge button">Organize <i
-            class="right arrow icon"></i></a>
+        {#        <a id="style_button" href="{% url "pages:search" %}" class="pages ui huge button"><i#}
+        {#                class="left arrow icon"></i> Compete</a>#}
+        {#        <a id="style_button" href="{% url "pages:organize" %}" class="pages ui huge button">Organize <i#}
+        {#                class="right arrow icon"></i></a>#}
     </div>
 
-    {#            <div class="ui middle aligned stackable grid container">#}
-    {#                <div class="row">#}
-    {#                    <div class="fourteen wide column centered">#}
-    {#                        <div class="ui fluid category search">#}
-    {#                            <div class="ui fluid icon input">#}
-    {#                                <input class="prompt" type="text" placeholder="Search machine learning...">#}
-    {#                                <i class="search icon"></i>#}
-    {#                            </div>#}
-    {#                            <div class="results fluid"></div>#}
-    {#                        </div>#}
-    {#                    </div>#}
-    {#                </div>#}
-    {#            </div>#}
 {% endblock %}
 
 {% block content %}
+    <div class="home-wrap">
+        {% if not user.is_authenticated %}
+            <div class="column-wrap">
+                <div class="vert-wrap">
+                    <div class="segment-container ui segment">
+                        <div class="ui header">
+                            Popular Competitions
+                        </div>
+                        <div class="container-content">
+                            <popular-competitions></popular-competitions>
```

Comment:
Popular competitions and featured competitions both hit the same endpoint but don't share data, so we have to ping that endpoint twice to populate data. And since it seems one of these doesn't appear without the other, we should probably combine these into a different riot tag like `front-page-competitions` or something. So we only hit the endpoint once and get the data for both. Alternatively we could pass the data around with riot observables, but just combining them into a single tag will be easier to track.

Reply:
Fixed for both popular and featured competitions